source/tests/system/nirfsg_driver_api_tests.cpp

```diff
@@ -3,8 +3,8 @@
 #include <thread>
 
 #include "device_server.h"
+#include "ivi.h"
```

Comment:
Why `ivi`? Is it for success like the RFSA comment says?

Only depending on ivi is definitely an improvement. Both because it is "less stuff" and because it is not likely to have the same gotchas as using the driver header. But, like the driver header, it would be in the "sever internals domain" not the "client api domain" and ... wouldn't be available to non-C clients.

[This feedback can be dropped].

Reply:
Yeah, it was basically for any VI types and some errors. But you're right, it's still not in line with the idea of keeping the server internals out of the tests.

So I removed the references to them and replaced the types with `google::protobuf` types where it made sense like we do in other tests.